mkdocs.yml

```diff
@@ -139,6 +139,13 @@ nav:
       - Regression Analysis:
         - statistics/regression/LinearRegression.md
 
+  - Data Science:
+      - data-science/index.md
+
+      - Data Preparation & Preprocessing:
+        - data-science/data/preparation.md
+        - data-science/data/preprocessors.md  # TODO: different transformations
```

Comment:
The preprocessor.md file is not contained in this pull request. I don't know, if this was on purpose or by accident? To due this fact, in the left menu 'None' will be shown

Reply:
My bad, I've removed the file from the navigation. It will be part of the next PR. 😄